src/domain/request/usecase/ExecuteReadyRequestsUseCase.spec.ts

```diff
@@ -21,27 +21,47 @@ describe('ExecuteReadyRequestsUseCase', () => {
     const requests = [];
 
     return {
-      getReadyRequests: () => [new Request('123', 'json(http://example.com).key1', Date.now() - oneMinuteMillis)],
+      getReadyRequests: () => [new Request('SOME_REQUEST_ID', 'json(http://example.com).key1', Date.now() - oneMinuteMillis)],
```

Comment:
Extract `SOME_REQUEST_ID ` as a constant variable rather than providing this multiple times as a string.